.github/workflows/base-component-descriptor.yaml

```diff
@@ -138,6 +155,27 @@ jobs:
             ref: ${{ github.ref }}
           EOF
 
+          if [ -n "${{ inputs.component-references }}" ]; then
```

Comment:
I think we might deduplicate the "catting", like so:

- set `fail_if_absent` (or similar) if input is given
- check existence (fail if absent and fail_if_absent (`exit 1`), else `exit 0

Admittedly, at least for the exit0-part, this would need to be put into a function (or guarded w/ another if), as we will still need to keep going.

However, this way, we could combine catting w/ appending (and thus avoid having to call append w/ empty list - which I think is a but clumsy)